atd-vzv/src/views/summary/CrashesByMode.js

```diff
@@ -352,6 +371,11 @@ const CrashesByMode = () => {
                                       );
                                     }
                                   )}
+                                  <hr className="my-0"></hr>
+                                  <p className={`h6 text-center my-1`}>
```

Comment:
I saw in the git history that you had a `pb-1` in here, and I think that it should come back if that looks right to you. ✨ It adds some grey space below the final row of cells that I think the regression that Frank mentioned was meant to address. The final row hard coded in the code is not the final row anymore with the addition of recent rows and this total row.

Also, super small detail but the braces and backticks can be replaced with double quotes. Purely stylistic and feel free to ignore!

Reply:
lol good catch, i'm throwing it back in! unfortunately it doesn't address the regression frank mentioned but it does add some grey space to the last row that makes it look a little more even compared to the others :sparkles:

Reply:
awesome! 🚀 I re-read my comment and was being really unclear at the end of yesterday. 😴 When I was talking about the "final row hard coded", I really meant there is a magic number used to assign a value to `paddingBottom` earlier in the code. I think that will lead to the fix. 🤞